rubocop-rspec.gemspec

```diff
@@ -36,8 +36,7 @@ Gem::Specification.new do |spec|
     'documentation_uri' => 'https://docs.rubocop.org/rubocop-rspec/'
   }
 
-  spec.add_runtime_dependency 'rubocop', '~> 1.0'
-  spec.add_runtime_dependency 'rubocop-ast', '>= 1.1.0'
```

Comment:
[RuboCop 1.19.0](https://rubygems.org/gems/rubocop/versions/1.19.0) depends on `rubocop-ast` `>= 1.9.1, < 2.0`, so this line is redundant.